.github/workflows/release.yml

```diff
@@ -36,7 +36,6 @@ jobs:
           echo "build --config=maven" >> .bazelrc
           echo "build:linux --config=toolchain" >> .bazelrc
           echo "build:linux --extra_toolchains=@llvm_toolchain//:cc-toolchain-x86_64-linux" >> .bazelrc
-          echo "build:macos  --xcode_version_config=//.github:host_xcodes" >> .bazelrc
```

Comment:
Couldn't we unify the macOS versions used in the test and release workflows?

Reply:
The reason we use different versions is supposedly improved backwards compatibility. Seems to be a good time to read up on how Windows and macOS handle this and unify the versions if there is no benefit to using an older one.

Reply:
Agreed. This is not a blocker to merging this PR.